Tests/DataPipeline/DataPipelineCompatibilityTests.swift

```diff
@@ -7,23 +7,29 @@ import XCTest
 
 private typealias SavedEvent = [String: Any]
 
-class DataPipelineCompatibilityTests: UnitTest {
+class DataPipelineCompatibilityTests: IntegrationTest {
```

Comment:
Updated the test to inherit from `IntegrationTest` instead for two reasons: 1) `DeviceInfoStub` is already available in `IntegrationTest`, and 2) since we are testing multiple aspects, such as the attachment of the device token and the emission of events, `IntegrationTest` seemed like the more suitable option for this test.